datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -785,7 +785,6 @@ create table if not exists withdrawals (
     asset bytea not null,
     status withdrawal_status not null,
     ref text not null,
-    expiry timestamp with time zone not null,
```

Comment:
I suppose really we should be adding this in another file as its a migration....though given were we are at I'm ok with letting this slide for now - unless we want to start testing the process?

Reply:
I think it’s fine for now, as we are anoit to tag the version which will go to mainnet next after this merge